frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -257,7 +258,14 @@ export const Receive = ({
                   <Dialog
                     open={!!(account && verifying)}
                     title={t('receive.verifyBitBox02')}
-                    disableEscape={true}
+                    // disable escape for secure outputs like the BitBox02, where the dialog is
+                    // dimissed by tapping the device
+                    disableEscape={verifying === 'secure'}
+                    onClose={() => {
+                      if (verifying === 'insecure') {
+                        setVerifying(false);
```

Comment:
This is delayed by 300ms because the Dialog component has this timeout before calling onClose. Seems like a bug. @thisconnect